core/src/main/scala/com/criteo/cuttle/Database.scala

```diff
@@ -119,6 +114,11 @@ private[cuttle] object Database {
 
     // Try to insert our lock at bootstrap
     (for {
+      _ <- sql"""CREATE TABLE IF NOT EXISTS locks (
```

Comment:
Is this related to your change? Or just some additional refactoring?

Because I don't understand how this impact the feature you expose. Here.

Reply:
Yep there are some kind of unrelated changes here, I added them in the commit message. Basically I'm locking the database while doing schema migration so we have to move the lock table creation out of the schema migration.